pubspec.yaml

```
@@ -7,21 +7,21 @@ funding:
 
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: '>=3.0.0 <4.0.0'
 
 dependencies:
   crypto_keys: ^0.3.0+1
   meta: ^1.1.6
   typed_data: ^1.0.0
   x509: ^0.2.1
-  http: ^0.13.0
+  http: ^1.0.0
```

```suggestion
  http: '>=0.13.0 <2.0.0'
```

To better support the migration!